RELEASING.md

```diff
@@ -69,10 +69,26 @@ For a discussion on whether to add a prefixing `v` to release tags, see [#35](ht
 
 Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v2.0.0`, as well as `patch` and `backport`. Use release labels to target an issue or a PR for a given release. See [MAINTAINERS](MAINTAINERS.md#triage-open-issues) for more information on triaging issues.
 
-## Releasing
+## Release Scheduling
 
-The OpenSearch release process is centralized.
+The OpenSearch release process is centralized and has several steps along the way.  As part of this process, the Engineering Efficiency team will cut a release ticket and assign an overall release manager for the release.  They will also cut individual tickets per repo for the release with what's expected to go in that release.   
 
+### Release Schedule Date Definitions:
+
+#### Feature freeze
+All code changes that add new capabilities to OpenSearch need to be merged to main and backported to the planned release. Additional merges/backports can happen for bugs fixes and CVEs.  Until we have all plugins using CI, we may continue to have an earlier feature freeze date for OpenSearch core and OpenSearch Dashboards.
+
+#### Code Freeze
+No more changes can be merged to the release branch after this point except in the case of blocking changes.  This generally happens 1 week before the release date for minor versions.
+
+#### Release Date
```

Comment:
I feel like we could use a ZBB, zero bug bounce, date. No need to make this change here, but it's generally a healthy date of "everything is working and there are no known issues that must be fixed in this release, but let's give it more time to find new bugs".

Reply:
I agree. It's totally fine to discover unknown bugs after code freeze but we should have fixed all known bugs at this point in time.

Reply:
I fleshed it out a bit more with the description of ZBB.  See if that scratches your itch.